project-base/tests/ShopBundle/Acceptance/acceptance/PageObject/Admin/ProductEditPage.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin;
+
+use Tests\ShopBundle\Acceptance\acceptance\PageObject\AbstractPage;
+
+class ProductEditPage extends AbstractPage
+{
+    /**
+     * @param int $productId
+     */
+    public function saveInvalidForm($productId)
```

Comment:
I would name this method more specifically - it saves form with invalid parameter...